aggregate/__main__.py

```diff
@@ -149,20 +165,67 @@ def main():
         project=config_values['PROJECT'],
     )
 
+    images = {}
     for function in functions:
-        # Create the function and it's corresponding pubsub and subscription.
-        fxn, _, _ = create_cloud_function(
-            name=function,
-            config_values=config_values,
-            service_account=config_values['GCP_SERVICE_ACCOUNT'],
-            pubsub_topic=pubsub,
-            cloud_service=cloud_service,
-            function_bucket=function_bucket,
-            source_archive_object=source_archive_object,
-            slack_channel=slack_channel,
-        )
+        if config_values.get('TYPE') == 'function':
+            # Create the function and it's corresponding pubsub and subscription.
+            create_cloud_function(
+                name=function,
+                config=config_values,
+                service_account=config_values['GCP_SERVICE_ACCOUNT'],
+                pubsub_topic=pubsub,
+                cloud_service=cloud_service,
+                function_bucket=function_bucket,
+                source_archive_object=source_archive_object,
+                slack_channel=slack_channel,
+            )
+        else:
+            _, image, full_image_name, _, _ = create_cloudrun_job(
+                name=function,
+                config=config_values,
+                service_account=config_values['GCP_SERVICE_ACCOUNT'],
+                pubsub_topic=pubsub,
+                cloudrun_service=cloudrun_service,
+                slack_channel=slack_channel,
+                prebuilt_images=images,
+            )
+            images[config_values['DOCKERIMAGE']] = (image, full_image_name)
+
+
+def configure_docker():
+    sp.check_output(['gcloud', 'auth', 'configure-docker'], stderr=sp.DEVNULL)
+
+
+def get_oauth_token():
+    result = sp.check_output(['gcloud', 'auth', 'print-identity-token'])
+    return result.decode('utf-8')
+
+
+def setup_docker_image(name, dockerfile, project):
+    # Get dockerfile folder
+    path = os.path.dirname(os.path.abspath(dockerfile))
+
+    image_name = f'gcr.io/{project}/{name}'
```

Comment:
That's not the registry we use for building images. We use an Artifact Registry in `australia-southeast1`, to avoid egress charges.

Reply:
Changed to push to the cpg-common artifact registry